.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -139,6 +139,8 @@ AddType text/vtt                            vtt
     # it should probably be best to combine the next few lines in one reg exp but for clarity keeping them separate
     # the first one redirects
     RewriteRule ^acs/?$ /acs/3.69/welcome/index.html [R=301]
+    # redirect to the latest release notes
+    RewriteRule ^acs/release_notes/$ /acs/3.69/release_notes/369-release-notes.html? [R=301,NE]
```

Comment:
@gaurav-nelson you could change "html?" to "html", unless there is an explicit reason to include the "?" for optional matching in the target URL. Sorry I didn't spot this on my first review.